pandas/tests/test_window.py

```diff
@@ -3288,6 +3288,132 @@ def test_min_periods(self):
         result = df.rolling('2s', min_periods=1).sum()
         tm.assert_frame_equal(result, expected)
 
+    def test_closed(self):
+
+        # closed=both only valid for datetimelike
```

Comment:
I would add some tests here that are the same eactly but include ``closed='right'`` (don't have to duplicate all, just a couple)